src/outgoing.rs

```diff
@@ -63,7 +65,9 @@ impl FromStr for Outgoing {
       .unwrap();
     }
 
-    Ok(if SATPOINT.is_match(s) {
+    Ok(if s.parse::<u64>().is_ok() {
```

Comment:
This should allow any representation of a sat. Have a look at the `Sat` type in `sat.rs`

Reply:
```
pub struct Sat(pub u64);
```

`s.parse::<u64>().is_ok()` is only for u64 representation , do you mean to include `Degree`, `Name`, or `Percentile` etc